lib/vanagon/platform.rb

```diff
@@ -239,6 +242,7 @@ def initialize(name) # rubocop:disable Metrics/AbcSize
       # Our first attempt at defining metadata about a platform
       @cross_compiled ||= false
       @valid_operators ||= ['<', '>', '<=', '>=', '=']
+      @servicetypes = []
```

Comment:
Is there a reason this isn't an `||=`?

Reply:
`@servicetypes` is not something you set directly, it's just an underlying datastructure, so we want to initialize it to an empty array, then other methods will set it.